modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -1951,7 +1951,13 @@ private void updateDisabled() {
      * specified selector, this function returns the first of them.
      * <p>
      *     For example, if a Node is given the id of "myId", then the lookup method can
-     *     be used to find this node as follows: <code>scene.lookup("#myId");</code>.
+     *     be used to find this node as follows: {@code scene.lookup("#myId");}.
+     * </p>
+     * <p>
+     *     If two nodes, lets say NodeA and NodeB have same style class "myStyle" and NodeA has
+     *     a pseudo state "myPseudo", then to find NodeA, the lookup method can be used as follows:
+     *     {@code scene.lookup(".myStyle:myPseudo");} or {@code scene.lookup(":myPseudo");}. If no pseudo class is specified
```

Comment:
The sentence about what happens when no pseudo class is specified is true in general and not really part of this example. I recommend moving it before the two examples. And I think it could be worded more clearly.

```
    * If the lookup selector does not specify a pseudo class, the lookup will ignore pseudo class
    *  states; it will return the first matching node even if it contains pseudo classes.
```

Reply:
Or possibly even more clear:

```
    * If the lookup selector does not specify a pseudo class, the lookup will ignore pseudo class
    * states; it will return the first matching node whether or not it contains pseudo classes.
```